.github/workflows/deploy-move-package.yml

```diff
@@ -0,0 +1,86 @@
+name: Deploy Move package
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Name of branch to checkout'
+        required: true
+      network:
+        description: 'Network to deploy to'
+        required: true
+      faucet:
+        description: 'TBD'
+        required: true
+      script-path:
+        description: 'Script to tun'
+        required: true
+      dry-run:
+        description: 'Run in dry-run mode'
+        type: boolean
+        required: false
+        default: true
+
+jobs:
+  deploy-move-package:
+    environment: release
+    name: TBD
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out the repo
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.inputs.branch }}
+
+    - name: Start iota sandbox
+      uses: './.github/actions/iota-rebase-sandbox/setup'
+      with:
+        platform: 'linux'
+        iota-version: ${{ github.event.inputs.network }}
+        start-sandbox: false
+
+    - name: Setup Keytool
+      uses: './.github/actions/iota-rebase-sandbox/load-keytool'
+      with:
+        network: ${{ github.event.inputs.network }}
+        pk: ${{secrets.TRUST_FRAMEWORK_PRODUCTS_PRIVATE_KEY}}
+        bak_pk: ${{secrets.TRUST_FRAMEWORK_PRODUCTS_PRIVATE_KEY_BACKUP}} #only for testing, TODO: remove
+        faucet: ${{ github.event.inputs.faucet }}
+
+    - name: Import GPG key
+      id: import-gpg
+      uses: crazy-max/ghaction-import-gpg@cb4264d3319acaa2bea23d51ef67f80b4f775013
+      with:
+        gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+        passphrase: ${{ secrets.GPG_PASSPHRASE }}
+        git_user_signingkey: true
+        git_commit_gpgsign: true
+        git_tag_gpgsign: true
+
+    - name: Run script
+      if: ${{ github.event.inputs.dry-run == 'false' }}
+      run: |
+        source ${{ github.event.inputs.script-path}}
+
+    - name: Commit changes
+      run: |
+        git add .
+        if [[ $(git diff --stat --staged) == '' ]]; then
+          echo 'repository unmodified'
+          exit 1
+        fi
+        git commit -m "bump version"
+        
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@67df31e08a133c6a77008b89689677067fef169e
+      with:
+        committer: GitHub <noreply@github.com>
+        author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+        branch: chore/deploy-${{ github.event.inputs.network }} 
+        delete-branch: true
+        title: 'Deploy ${{ github.event.inputs.network }}'
```

Comment:
Should the PR title be deploy related or refer to the version bump like the commit?

Reply:
good question 🤔